chemprop/v2/data/utils.py

```diff
@@ -1,66 +1,173 @@
-from collections import defaultdict
-from typing import Sequence
+import copy
+import itertools
+import logging
+from enum import auto
+from typing import Sequence, Tuple
 
 import numpy as np
-from rdkit.Chem.Scaffolds import MurckoScaffold
-
+from astartes import train_test_split, train_val_test_split
+from astartes.molecules import train_test_split_molecules, train_val_test_split_molecules
 
+from chemprop.v2.cli.utils import LOG_DIR, NOW
 from chemprop.v2.data.datapoints import MoleculeDatapoint
+from chemprop.v2.data.datasets import MoleculeDataset
+from chemprop.v2.data.utils import log_scaffold_stats
+from chemprop.v2.utils.utils import AutoName
+
+logger = logging.getLogger(__name__)
+
+DEFAULT_LOGFILE = f"{LOG_DIR}/{NOW}.log"
+LOG_LEVELS = [logging.ERROR, logging.WARNING, logging.INFO, logging.DEBUG]
+SUBCOMMANDS = []
+
+
+class SplitType(AutoName):
+    CV_NO_VAL = auto()
+    CV = auto()
+    SCAFFOLD_BALANCED = auto()
+    RANDOM_WITH_REPEATED_SMILES = auto()
+    RANDOM = auto()
+    KENNARD_STONE = auto()
+    KMEANS = auto()
 
 
 def split_data(
     data: Sequence[MoleculeDatapoint],
-    split: str = "random",
-    sizes: tuple[float, float, float] = (0.8, 0.1, 0.1),
-    k: int = 5,
-    fold: int = 0,
-):
+    split: str | SplitType,
```

Comment:
why was the default of `random` removed?

Reply:
on accident, adding back